marathon-interface/src/main/scala/mesosphere/marathon/interface/PathId.scala

```diff
@@ -0,0 +1,23 @@
+package mesosphere.marathon.interface
```

Comment:
We already talked about that, but in order to keep track:
I really much like that package name – but how about Java interoperability?